src/evaluationSummaryAggregator.ts

```diff
@@ -0,0 +1,110 @@
+// TODO: flush on beforeunload
+
+// TODO: should we retry the data chunk if a flush fails?
+
+// TODO: pause when offline?
+
+// TODO: flush when we receive a config update (or as a result of a context update...but that should trigger a config update anyway)
+
+import { PeriodicSync } from "./periodicSync";
+import { Config, ConfigEvaluationMetadata } from "./config";
+import { type prefab } from "./prefab";
+
+type ConfigEvaluationCounter = Omit<ConfigEvaluationMetadata, "type"> & {
+  selectedValue: any;
```

Comment:
@semanticart I changed this to `any` so that I don't have to map out the possible types here. See line 59 where `selectedValue` gets constructed from the config type string and config value string that is sent down with the evaluated configs.

Reply:
@ayip8 can we use `unknown` instead of `any`?  `any` can have some unintended propagation but `unknown` enforces stricter type checking and requires narrowing.

Reply:
If that ends up just pushing the headache around, an `any` is acceptable.